app/controllers/box_request_triage_controller.rb

```diff
@@ -54,6 +55,8 @@ def create
 
     box_request.save!
 
+    render json: { "redirect_url": box_request_thank_you_path },
+           status: 200
```

Comment:
It occurs to me that this should probably be a 302. Or is that not how React redirection works?